cli/command/container/cp.go

```diff
@@ -34,11 +39,70 @@ const (
 type cpConfig struct {
 	followLink bool
 	copyUIDGID bool
+	quiet      bool
 	sourcePath string
 	destPath   string
 	container  string
 }
 
+//copyReader is a Wrapper to content
+type copyReader struct {
+	io.ReadCloser
+	toContainer   bool
+	total         int64
+	copySize      int64
+	containerName string
+	copyPath      string
+	quiet         bool
+}
+
+func updateArrowLoading(value float64) string {
+	division := int(value) / 10
+	newArrow := "["
+	i := 0
+	j := 0
+
+	for i < division {
+		newArrow += "="
+		i++
+	}
+
+	if i > 0 && i < 10 {
+		newArrow += ">"
+		i++
+	}
+
+	for j < 10-i {
+		newArrow += "_"
+		j++
+	}
+
+	newArrow += "]"
+
+	return newArrow
+
+}
+
+func (pt *copyReader) Read(p []byte) (int, error) {
+	n, err := pt.ReadCloser.Read(p)
+	pt.total += int64(n)
+
+	percent := float64(pt.total) / float64(pt.copySize)
+
+	arrow := updateArrowLoading(percent * 100)
+
+	if err == nil && !pt.quiet {
+		fmt.Print("\033[u\033[K")
+		if pt.toContainer {
+			fmt.Printf("Copying to container %s %s/%s", arrow, units.HumanSize(float64(pt.total)), units.HumanSize(float64(pt.copySize)))
```

Comment:
I think the `copyReader` should take an i/o stream instead of assuming os.Stdout (As `Printf` and `Println` do).
Then we need `fmt.Fprint[ln,f]`.

Reply:
Right 😉

Reply:
@cpuguy83 how we can sinalize to `copyReader` the right output (os.Stdout, file, etc...)?

Reply:
This can be passed down froM the cli object.

Reply:
Oh, right!

Reply:
Hi @cpuguy83 , I made the change in the output, can you review?